sbi/inference/posteriors/score_posterior.py

```diff
@@ -271,9 +271,6 @@ def log_prob(
         theta: Tensor,
         x: Optional[Tensor] = None,
         track_gradients: bool = False,
-        atol: float = 1e-5,
```

Comment:
Instead of removing these altogether, should we not pass them to `potential_fn.set_x` instead, so we can still specify the tolerance when computing the log probs?

Reply:
Thanks!